### PR TITLE
Minor changes in C++ source code

### PR DESCRIPTION
**Pull request by RyDroid**

### CODE REVIEW

File: src/engine/video/image_base.cpp [hunk 1]
```diff
@@ -383,12 +394,14 @@ void ImageMemory::CopyFromImage(BaseTexture *img)
...
-        PRINT_ERROR << "Failed to malloc enough memory to copy the image" << std::endl;
+        PRINT_ERROR << "Failed to malloc enough memory to copy the image"
+                    << std::endl;
         return;
     }
 
     for (uint32_t i = 0; i < img->height; ++i) {
-        std::vector<uint8_t>::const_iterator start = _pixels.begin() + i * src_bytes + src_offset;
+        std::vector<uint8_t>::const_iterator start =
+          _pixels.begin() + i * src_bytes + src_offset;
```
Comment (Bertram25) [inline src/engine/video/image_base.cpp]: 2 spaces.

File: src/engine/video/image_base.cpp [hunk 2]
```diff
@@ -454,8 +468,10 @@ void ImageMemory::VerticalFlip()
     flipped.reserve(_pixels.size());
 
     for (uint32_t i = 1; i <= _height; ++i) {
-        std::vector<uint8_t>::const_iterator start = _pixels.end() - (i * _width * GetBytesPerPixel());
-        std::vector<uint8_t>::const_iterator end = start + _width * GetBytesPerPixel();
+        std::vector<uint8_t>::const_iterator start =
+          _pixels.end() - (i * _width * GetBytesPerPixel());
```
Comment (Bertram25) [inline src/engine/video/image_base.cpp]: 2x 2 spaces

File: src/engine/video/image_base.cpp [hunk 3]
```diff
@@ -527,41 +545,47 @@ bool BaseTexture::RemoveReference()
...
+    }
+    return ref_count == 0;
 }
 
 // -----------------------------------------------------------------------------
 // ImageTexture class
 // -----------------------------------------------------------------------------
 
-ImageTexture::ImageTexture(const std::string &filename_, const std::string &tags_, int32_t width_, int32_t height_) :
+ImageTexture::ImageTexture(const std::string &filename_,
```
Comment (Bertram25) [inline src/engine/video/image_base.cpp]: Btw, just to point it out. this is bad parameter naming. ;)

File: src/engine/video/text.cpp [hunk 4]
```diff
@@ -248,13 +253,14 @@ bool TextTexture::Regenerate()
...
-    if(TextManager->_RenderText(string, style, buffer) == false)
+    if(!TextManager->_RenderText(string, style, buffer))
         return false;
 
     width = buffer.GetWidth();
     height = buffer.GetHeight();
 
-    TexSheet *sheet = TextureManager->_InsertImageInTexSheet(this, buffer, true);
+    TexSheet *sheet =
+      TextureManager->_InsertImageInTexSheet(this, buffer, true);
```
Comment (Bertram25) [inline src/engine/video/text.cpp]: 2 spaces

File: src/engine/video/text.cpp [hunk 5]
```diff
@@ -699,16 +714,20 @@ bool TextSupervisor::LoadFonts(const std::string& locale_name)
...
+        PRINT_ERROR << "Attempted to load a text style of size zero: "
+                    << textstyle_name << std::endl;
         return false;
     }
 
     // Check whether the TextStyle name is not already taken
     bool reload = false;
-    std::map<std::string, FontProperties *>::iterator it = _font_map.find(textstyle_name);
+    std::map<std::string, FontProperties *>::iterator it =
+      _font_map.find(textstyle_name);
```
Comment (Bertram25) [inline src/engine/video/text.cpp]: 2 spaces

File: src/engine/video/text.cpp [hunk 6]
```diff
@@ -756,9 +777,11 @@ bool TextSupervisor::_LoadFont(const std::string& textstyle_name, const std::str
 
 void TextSupervisor::_FreeFont(const std::string &font_name)
 {
-    std::map<std::string, FontProperties*>::iterator it = _font_map.find(font_name);
+    std::map<std::string, FontProperties*>::iterator it =
+      _font_map.find(font_name);
```
Comment (Bertram25) [inline src/engine/video/text.cpp]: 2 spaces

**Comment by Bertram25** [comment]
Seems you're making yourself comfortable with the code. ;)

File: src/engine/video/image_base.cpp (hunk 1 above)
Comment (RyDroid) [inline src/engine/video/image_base.cpp]: More?

File: src/engine/video/image_base.cpp (hunk 2 above)
Comment (RyDroid) [inline src/engine/video/image_base.cpp]: Do you want 4 spaces instead of 2?

File: src/engine/video/image_base.cpp (hunk 1 above)
Comment (Bertram25) [inline src/engine/video/image_base.cpp]: yep, let's 4 as much as possible, except for alignment as you've done in a few places.

File: src/engine/video/image_base.cpp (hunk 2 above)
Comment (Bertram25) [inline src/engine/video/image_base.cpp]: yep, if possible.

**Comment by Bertram25** [comment]
I'll fix this and merge (keeping your authorship).

**Comment by Bertram25** [comment]
Merged as 1aae7b921fd57583b801b25f2acc02c06efde17d

**Comment by RyDroid** [comment]
Ok, great.

**Pull request closed without merging**